src/omnibase_core/models/examples/model_environment_properties.py

```
@@ -60,21 +60,21 @@ def get_typed_value(self, key: str, expected_type: type[T], default: T) -> T:
         try:
             # Use ModelPropertyValue's type-safe accessors based on expected type
             if expected_type == str:
```

Ruff (0.14.7)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)